app/components/booking/form.tsx

```diff
@@ -377,7 +379,9 @@ export function BookingForm({
         </div>
         {isNewBooking ? (
           <div className="text-right">
```

Comment:
You need to also add some hidden inputs to send the `assetIds` to the action using the formData.
Here is some pseudo code:
```ts

          {assetIds.map((item, i) => (
            <input
              key={item}
              type="hidden"
              name={`assetIds[${i}]`}
              value={item}
            />
          ))}
          ```

Reply:
Added a hidden input. Resolved!